libs/island-ui/core/src/lib/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,37 +1,78 @@
-import React, { FC, Children } from 'react'
+import React, { FC, ReactNode } from 'react'
 import { Text } from '../Text/Text'
-import { Icon } from '../Icon/Icon'
+import { Tag } from '../Tag/Tag'
+import { TagVariant } from '../Tag/Tag'
 import { Colors } from '@island.is/island-ui/theme'
+import cn from 'classnames'
 import * as styles from './Breadcrumbs.treat'
 
+export interface BreadCrumbItem {
+  title: string
+  href?: string
+  slug?: string[]
+  typename?: string
+  isTag?: boolean
+}
+
 interface BreadcrumbsProps {
+  items: BreadCrumbItem[]
   label?: string
-  color?: Colors
+  color?: keyof typeof styles.breadcrumb
+  tagVariant?: TagVariant
   separatorColor?: Colors
+  renderLink?: (link: ReactNode, item: BreadCrumbItem) => ReactNode
 }
 
 export const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  items,
   label = 'breadcrumb',
   color = 'blue400',
-  separatorColor = 'blue400',
-  children,
+  tagVariant = 'blue',
+  renderLink = (link) => link,
 }) => {
-  const crumbs = Children.toArray(children).filter((c) => c)
-
   return (
-    <div aria-label={label}>
-      {crumbs.map((child, index) => (
-        <span key={index}>
-          <Text variant="eyebrow" as="span" color={color}>
-            {child}
+    <div aria-label={label} className={styles.wrapper}>
+      {items.map((item, index) => {
+        const isLink: boolean = !!item.href || !!item.slug
+        const renderCrumb = item.isTag ? (
+          <Tag disabled={!isLink} variant={tagVariant}>
+            {item.title}
+          </Tag>
+        ) : (
+          <Text as="span" variant={'eyebrow'} color={color}>
+            {item.title}
           </Text>
-          {crumbs.length - 1 > index && (
-            <span className={styles.divider}>
-              <Icon type="bullet" width="4" color={separatorColor} />
-            </span>
-          )}
-        </span>
-      ))}
+        )
+
+        return (
+          <span key={index} className={styles.wrapper}>
+            {renderLink(
+              isLink ? (
+                <a
+                  href={item?.href}
+                  tabIndex={item.isTag ? -1 : undefined}
+                  style={{
+                    textDecoration: item.isTag ? 'none' : undefined,
+                    display: 'inline-flex',
+                  }}
+                  className={cn(styles.breadcrumb[color], {
+                    [styles.focusable[color]]: isLink,
+                  })}
+                >
+                  {renderCrumb}
+                </a>
+              ) : (
+                renderCrumb
```

Comment:
shouldn't this be outside `renderLink` function? same question regarding `isLink` if you use renderLink to wrap link with router, then this would probably get wrapped?